src/help.raw

```diff
@@ -94,6 +94,7 @@ OPTIONS
      --min-dimension WxH   Only show images with width >= W and height >= H
      --max-dimension WxH   Only show images with width <= W and height <= H
      --scroll-step COUNT   scroll COUNT pixels when movement key is pressed
+     --inotify             automatically reload shown image if file was changed
```

Comment:
I'm all for this feature; now given that the exact API is an implementation detail, should this maybe called `--automatic-reload`, or something that describes the intent?

Reply:
good point! just changed the option to `--auto-reload`.